server/controllers/client.ts

```diff
@@ -147,18 +137,6 @@ async function generateVideoChannelHtmlPage (req: express.Request, res: express.
   return sendHTML(html, res)
 }
 
-async function generateManifest (req: express.Request, res: express.Response) {
-  const manifestPhysicalPath = join(root(), 'client', 'dist', 'manifest.webmanifest')
-  const manifestJson = await fs.readFile(manifestPhysicalPath, 'utf8')
-  const manifest = JSON.parse(manifestJson)
-
-  manifest.name = CONFIG.INSTANCE.NAME
-  manifest.short_name = CONFIG.INSTANCE.NAME
-  manifest.description = CONFIG.INSTANCE.SHORT_DESCRIPTION
-
```

Comment:
You may add a hook here to override the all manifest and build your own manifest in a plugin, it will be very lighter to build though you won't have to re-build all the PeerTube app.

It will also allow to not touch the current behaviour.